specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/onPremiseIotSensors.json

```diff
@@ -0,0 +1,276 @@
+          },
+          {
+            "$ref": "#/parameters/OnPremiseIotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Updated on-premise IoT sensor",
+            "schema": {
+              "$ref": "#/definitions/OnPremiseIotSensor"
+            }
+          },
+          "201": {
+            "description": "Created on-premise IoT sensor",
+            "schema": {
+              "$ref": "#/definitions/OnPremiseIotSensor"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete on-premise IoT sensor": {
+            "$ref": "./examples/OnPremiseIotSensors/Delete.json"
+          }
+        },
+        "tags": [
+          "On-Premise IoT Sensors"
+        ],
+        "operationId": "OnPremiseIotSensors_Delete",
+        "description": "Delete on-premise IoT sensor",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/OnPremiseIotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted on-premise IoT sensor"
+          },
+          "204": {
+            "description": "On-premise IoT sensor does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Security/onPremiseIotSensors/{onPremiseIotSensorName}/downloadActivation": {
+      "post": {
+        "x-ms-examples": {
+          "Download activation file": {
+            "$ref": "./examples/OnPremiseIotSensors/DownloadActivation.json"
+          }
+        },
+        "tags": [
+          "On-Premise IoT Sensors"
+        ],
+        "operationId": "OnPremiseIotSensors_DownloadActivation",
+        "description": "Download sensor activation file",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/OnPremiseIotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Downloaded activation file"
+          },
```

Comment:
What is the schema of the response? I assume as a "download" there had to be something in response.

Reply:
Actual file download (binary).

Reply:
But you need to specify it somehow? Else how SDK knows that there is a binary file as reponse?

Usually it is 
```
            "schema": {
              "type": "file"
            }
```

Reply:
Nice! Pushed fix.